native/acter/src/api/room.rs

```diff
@@ -1243,18 +1225,12 @@ impl Room {
                             (request, filename)
                         }
                         MessageType::Video(content) => {
+                            let format = into_media_format(Some(thumb_size));
```

Comment:
Suggestions above makes this smaller and only if actually a source is found by doing:
```suggestion
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1246/commits/518193fa236e942607e738383ba8ffb64b96b9ca